### ISSUE TITLE
Georgia Tech for Emory Items - Issue with display label codes

### ISSUE THREAD

**Issue by libdgg** [emory-libraries/blacklight-catalog#1174]
On go-live GT reported seeing display label codes "c.uresolver..." when finding/requesting Emory items. Heather Jeffcoat (GT) and Doug met with Efi on zoom. Efi identified this as a setting (that is turned on as part of the GO-VE steps). Efi turned it off and the issue was resolved for GT primo. Emory item/requesting was not affected. Chris Helms (GT) provided Efi with a GT test user account. Efi and his team will be doing further tests starting Sunday. We should follow up with them next week.

**Comment by libdgg** [comment]
Notes from the GT/Emory meeting 2022-01-06
https://emory-my.sharepoint.com/:w:/r/personal/lmeyer2_emory_edu/_layouts/15/Doc.aspx?sourcedoc=%7BAB99FBA6-F986-430B-9494-BA5FE7F80FD0%7D&file=2022-01-06-Emory-Library-Search-Go-Live-Check-In.docx&action=default&mobileredirect=true&cid=7eda44c7-7f57-4d12-ab02-b167773deb18

**Comment by libdgg** [comment]
I'll check basecamp and email on Monday to see if there is an update or to ping Efi about next steps.

**Comment by tmiles2** [comment]
@libdgg Can you provide me access to the linked document?

**Comment by libdgg** [comment]
@tmiles2 you should have access now. please let me know otherwise. thanks

**Comment by tmiles2** [comment]
@libdgg Thanks I have access.

**Comment by libdgg** [comment]
Efi provided the following update. I responded to see if we need to take any other steps at this time. I emailed this update to the LSC Technology Subcommittee so GaTech knows as well.
"Hi Doug, Indeed, the parameter we switched off responsible on changing code tables on the old Primo environment and as a result we seen the label changes in the GT Primo.
We will keep this parameter off until GT go Live with Primo VE as well.
Regards, Efi"
